build.gradle

```diff
@@ -115,10 +115,6 @@ subprojects {
     options.encoding = 'UTF-8'
   }
 
-  ext {
-    jmhVersion = '1.21'
```

Comment:
Somehow, this did not seem to have any effect. I had to move it to the jmh block.